src/Security/Handler/RoleSecurityHandler.php

```diff
@@ -125,4 +116,52 @@ private function isAnyGranted(array $attributes, ?object $subject = null): bool
 
         return false;
     }
+
+    /**
+     * @param array<string|Expression> $attributes
+     */
+    private function hasAllRule(mixed $attributes): bool
```

Comment:
Is "rule" the right term here?

Reply:
Thanks for the review. Adjusted the name